3-UsingAPIs/Week1/homework/ex2-checkDoubleDigits.js

```diff
@@ -12,8 +12,14 @@ Complete the function called `checkDoubleDigits` such that:
   "Expected a double digit number but got `number`", where `number` is the 
   number that was passed as an argument.
 ------------------------------------------------------------------------------*/
-function checkDoubleDigits(/* TODO add parameter(s) here */) {
-  // TODO complete this function
+function checkDoubleDigits(digit) {
+  return new Promise((resolve, reject) => {
+    if (digit > 9 && digit < 100) {
```

Comment:
```suggestion
    if (digit >= 10 && digit <= 99) {
```

Since we are talking in terms of double-digit numbers I would be inclined to express the bounds in the same terms. Just my personal preference, but your code is fine too, of course.